### PR TITLE
Add documentation for cloud optics

### LINKED ISSUE #332: Add / Clean up equations for cloud optics

### LINKED ISSUE #333: Add documentation for cloud overlapping assumption

### PR DESCRIPTION
**Pull request by szy21**
## Purpose 

Closes #332  
Closes #333 

## Content

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

File: docs/src/Optics.md [hunk 1]
```diff
@@ -89,3 +89,31 @@ For longwave bands, the Planck fraction (defined as the fraction of the band-int
 
 ### Lookup tables
 The lookup tables for the spectral map between ``\nu`` and ``g`` are stored in netcdf files [here](https://caltech.box.com/shared/static/wbtrwp44dyn08g7mozjf4fcyrexwbe6a.gz). Longwave and shortwave lookup tables are stored in `clearsky_lw.nc` and `clearsky_sw.nc`, respectively. `LookUpLW` and `LookUpSW` read the lookup tables. The tabulated information includes the spectral distretization (bands), the major and minor gases considered in each band, and tabulated data of absorption coefficients, Planck fraction, and band-integrated Planck function. The absorption coefficients and Planck fraction are computed at pressures ``1 \mathrm{Pa} ≤ p ≤ 109663 \mathrm{Pa}`` in increments of ``\log(p/1 \mathrm{Pa}) = 0.2``, temperatures ``160 \mathrm{K} ≤ T ≤ 355 \mathrm{K}`` in ``15 \mathrm{K}`` increments, and ``0 ≤ \eta ≤ 1`` in ``1/8`` increments. The band-integrated Plack function is computed at temperatures ``160 \mathrm{K} ≤ T ≤ 355 \mathrm{K}`` in ``1 \mathrm{K}`` increments. A more detailed description of the spectral structure of the lookup tables can be found in `Appendix A` in [pincus2019](@cite).
```
Comment (tapios) [inline docs/src/Optics.md:91]:
```suggestion
The lookup tables for the spectral map between ``\nu`` and ``g`` are stored in netcdf files [here](https://caltech.box.com/shared/static/wbtrwp44dyn08g7mozjf4fcyrexwbe6a.gz). Longwave and shortwave lookup tables are stored in `clearsky_lw.nc` and `clearsky_sw.nc`, respectively. `LookUpLW` and `LookUpSW` read the lookup tables. The tabulated information includes the spectral discretization (bands), the major and minor gases considered in each band, and tabulated data of absorption coefficients, Planck fraction, and band-integrated Planck function. The absorption coefficients and Planck fraction are computed at pressures ``1 \mathrm{Pa} ≤ p ≤ 109663 \mathrm{Pa}`` in increments of ``\log(p/1 \mathrm{Pa}) = 0.2``, temperatures ``160 \mathrm{K} ≤ T ≤ 355 \mathrm{K}`` in ``15 \mathrm{K}`` increments, and ``0 ≤ \eta ≤ 1`` in ``1/8`` increments. The band-integrated Plack function is computed at temperatures ``160 \mathrm{K} ≤ T ≤ 355 \mathrm{K}`` in ``1 \mathrm{K}`` increments. A more detailed description of the spectral structure of the lookup tables can be found in `Appendix A` in [pincus2019](@cite).
```

Comment (tapios) [reply docs/src/Optics.md:91]: Also in l. 42 and 52 above, please replace "sin" by "\sin".

File: docs/src/Optics.md [hunk 2]
```diff
@@ -89,3 +89,31 @@ For longwave bands, the Planck fraction (defined as the fraction of the band-int
...
+
+!!! note
+    Absorption by both major and minor species is treated separately in the upper and lower atmosphere, defined as those pressures above and below the pressure defined in `press_ref_trop` in the lookup tables (~9948 Pa). This corresponds to element 13 in array `press_ref`. There is a single large table of absorption coefficients. Table elements 1:13 in the pressure dimension correspond to pressures >= `press_ref_trop` while table elements 14:60 correspond to pressures <= `press_ref_trop`. Elements 13 and 14 refer to the same pressure but for the lower and upper atmosphere respectively. This is why the the absorption coefficients lookup table have a pressure dimension of 60, but there are only 59 reference pressure levels in `press_ref`.
+
+## Cloud optics
+
+!!! note
+    Cloud optics can be calculated using either lookup tables or Pade approximates. Only the lookup table method is documented here.
```
Comment (tapios) [inline docs/src/Optics.md:99]:
```suggestion
    Cloud optics can be calculated using either lookup tables or Pade approximations. Only the lookup table method is documented here.
```

**Review by tapios — APPROVED** [review]
Thanks. LGTM.

File: docs/src/Optics.md [hunk 3]
```diff
@@ -89,3 +89,31 @@ For longwave bands, the Planck fraction (defined as the fraction of the band-int
...
+
+!!! note
+    Absorption by both major and minor species is treated separately in the upper and lower atmosphere, defined as those pressures above and below the pressure defined in `press_ref_trop` in the lookup tables (~9948 Pa). This corresponds to element 13 in array `press_ref`. There is a single large table of absorption coefficients. Table elements 1:13 in the pressure dimension correspond to pressures >= `press_ref_trop` while table elements 14:60 correspond to pressures <= `press_ref_trop`. Elements 13 and 14 refer to the same pressure but for the lower and upper atmosphere respectively. This is why the the absorption coefficients lookup table have a pressure dimension of 60, but there are only 59 reference pressure levels in `press_ref`.
+
+## Cloud optics
+
+!!! note
```
[the remark this answers is not in the document]
Comment (szy21) [reply docs/src/Optics.md:98]: It looks like this. See here for example: https://clima.github.io/RRTMGP.jl/previews/PR340/Optics/#Cloud-optics
Screenshot 2023-02-28 at 8 54 52 AM

[the remark this answers is not in the document]
Comment (tapios) [reply docs/src/Optics.md:98]: Oh, ok. That's good. It didn't render on the preview.

**Pull request merged**